math/caffe/Portfile

```diff
@@ -3,9 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        BVLC caffe 6d92d8fcfe0eea9495ffbc326256ec5b70c3eed1
-version             20150717
-revision            6
+name                caffe
+github.setup        BVLC caffe 1de4cebfb81d50267d0d8c2595372b14e1408248
+version             20170823
+revision            7
```

Comment:
`revision` should be cleared when the port is updated to a newer version.

Reply:
Removing it, missed that part from the documentation, Thanks! 😄